{
  "uid" : "54cce65e3ada6f01",
  "name" : "2.6 POST Create user with None name",
  "fullName" : "tests.create_users_test.TestCreateUser#test_post_create_user_with_none_name",
  "historyId" : "cbd2c4fa31e20d377255f92bbd065683",
  "time" : {
    "start" : 1708420827924,
    "stop" : 1708420828169,
    "duration" : 245
  },
  "status" : "skipped",
  "statusMessage" : "XFAIL Bag\n\nAssertionError: Unexpected status code. Expected not 201. Actual: 201",
  "statusTrace" : "self = <tests.create_users_test.TestCreateUser object at 0x7fb4acfc5ad0>\n\n    @pytest.mark.negative\n    @pytest.mark.xfail(reason='Bag')\n    @allure.title('2.6 POST Create user with None name')\n    def test_post_create_user_with_none_name(self):\n        response = BaseRequests.post(url='api/users', headers=self.data.CREATE_USER)\n>       Assertions.assert_is_not_code_status(response, self.status.STATUS_CODE_201)\n\ntests/create_users_test.py:91: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [201]>, expected_status_code = 201\n\n    @staticmethod\n    def assert_is_not_code_status(response: Response, expected_status_code):\n        \"\"\"Check status code is wrong\"\"\"\n        actual_status_code = response.status_code\n        with allure.step(f'Expected status is not equal {expected_status_code}'):\n>           assert actual_status_code != expected_status_code, \\\n                f'Unexpected status code. Expected not {expected_status_code}. Actual: {actual_status_code}'\nE           AssertionError: Unexpected status code. Expected not 201. Actual: 201\n\nbase/assertions.py:27: AssertionError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "_session_faker",
    "time" : {
      "start" : 1708420823646,
      "stop" : 1708420823661,
      "duration" : 15
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "testStage" : {
    "status" : "skipped",
    "statusMessage" : "XFAIL Bag\n\nAssertionError: Unexpected status code. Expected not 201. Actual: 201",
    "statusTrace" : "self = <tests.create_users_test.TestCreateUser object at 0x7fb4acfc5ad0>\n\n    @pytest.mark.negative\n    @pytest.mark.xfail(reason='Bag')\n    @allure.title('2.6 POST Create user with None name')\n    def test_post_create_user_with_none_name(self):\n        response = BaseRequests.post(url='api/users', headers=self.data.CREATE_USER)\n>       Assertions.assert_is_not_code_status(response, self.status.STATUS_CODE_201)\n\ntests/create_users_test.py:91: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [201]>, expected_status_code = 201\n\n    @staticmethod\n    def assert_is_not_code_status(response: Response, expected_status_code):\n        \"\"\"Check status code is wrong\"\"\"\n        actual_status_code = response.status_code\n        with allure.step(f'Expected status is not equal {expected_status_code}'):\n>           assert actual_status_code != expected_status_code, \\\n                f'Unexpected status code. Expected not {expected_status_code}. Actual: {actual_status_code}'\nE           AssertionError: Unexpected status code. Expected not 201. Actual: 201\n\nbase/assertions.py:27: AssertionError",
    "steps" : [ {
      "name" : "Expected status is not equal 201",
      "time" : {
        "start" : 1708420828169,
        "stop" : 1708420828169,
        "duration" : 0
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: Unexpected status code. Expected not 201. Actual: 201\n",
      "statusTrace" : "  File \"/home/runner/work/reqres/reqres/base/assertions.py\", line 27, in assert_is_not_code_status\n    assert actual_status_code != expected_status_code, \\\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Create User"
  }, {
    "name" : "tag",
    "value" : "@pytest.mark.xfail(reason='Bag')"
  }, {
    "name" : "tag",
    "value" : "negative"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "create_users_test"
  }, {
    "name" : "subSuite",
    "value" : "TestCreateUser"
  }, {
    "name" : "host",
    "value" : "fv-az693-149"
  }, {
    "name" : "thread",
    "value" : "2233-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.create_users_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 0,
        "skipped" : 6,
        "passed" : 0,
        "unknown" : 0,
        "total" : 6
      },
      "items" : [ {
        "uid" : "7a325779832a954b",
        "reportUrl" : "https://Nat754.github.io/reqres/8//#testresult/7a325779832a954b",
        "status" : "skipped",
        "statusDetails" : "XFAIL Bag\n\nAssertionError: Unexpected status code. Expected not 201. Actual: 201",
        "time" : {
          "start" : 1708420747761,
          "stop" : 1708420748008,
          "duration" : 247
        }
      }, {
        "uid" : "30254f13497167fd",
        "reportUrl" : "https://Nat754.github.io/reqres/7//#testresult/30254f13497167fd",
        "status" : "skipped",
        "statusDetails" : "XFAIL Bag\n\nAssertionError: Unexpected status code. Expected not 201. Actual: 201",
        "time" : {
          "start" : 1708418354136,
          "stop" : 1708418354501,
          "duration" : 365
        }
      }, {
        "uid" : "d6f2d15de45c6ef0",
        "reportUrl" : "https://Nat754.github.io/reqres/6//#testresult/d6f2d15de45c6ef0",
        "status" : "skipped",
        "statusDetails" : "XFAIL Bag\n\nAssertionError: Unexpected status code. Expected not 201. Actual: 201",
        "time" : {
          "start" : 1708412186499,
          "stop" : 1708412186740,
          "duration" : 241
        }
      }, {
        "uid" : "101f6cd2dc99a580",
        "reportUrl" : "https://Nat754.github.io/reqres/5//#testresult/101f6cd2dc99a580",
        "status" : "skipped",
        "statusDetails" : "XFAIL Bag\n\nAssertionError: Unexpected status code. Expected not 201. Actual: 201",
        "time" : {
          "start" : 1708388623747,
          "stop" : 1708388623997,
          "duration" : 250
        }
      }, {
        "uid" : "a630d867275160f",
        "reportUrl" : "https://Nat754.github.io/reqres/4//#testresult/a630d867275160f",
        "status" : "skipped",
        "statusDetails" : "XFAIL Bag\n\nAssertionError: Unexpected status code. Expected not 201. Actual: 201",
        "time" : {
          "start" : 1708385919291,
          "stop" : 1708385919645,
          "duration" : 354
        }
      } ]
    },
    "tags" : [ "negative", "@pytest.mark.xfail(reason='Bag')" ]
  },
  "source" : "54cce65e3ada6f01.json",
  "parameterValues" : [ ]
}